smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -72,7 +72,7 @@ def authPage() {
 		log.debug "have LIFX access token"
 
 		def options = locationOptions() ?: []
-		def count = options.size()
+		def count = options?.size() > 0 ? options.size()  : 0
```

Comment:
I just verified that this change indeed will not fix the problem. Line 74 guarantees that `count` will never be null. The problem is that the translation service is improperly checking the numeric value `count` and is interpreting a 0 value as not being set. We'll want to fix that in the platform (DM or the translation service) but for now this hack will get around the problem:
```
def count = options.size().toString()
```